x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -435,6 +444,15 @@ private ByteRange mapSubRangeToRegion(ByteRange range, int region) {
         );
     }
 
+    // package private for tests
+    int computeCacheFileRegionSize(long fileLength, int region) {
+        if (useFullRegionSize) {
+            return getRegionSize();
+        }
+        // the size of the region is computed from the file length
+        return getRegionSize(fileLength, region);
```

Comment:
Nit: It is a bit confusing that this method is also called `getRegionSize` which is completely different from the getter method `getRegionSize()`. I wonder whether we could inline it into this new method.

Reply:
Marking comment as resolved since this is now obsolete